packages/mgt-components/src/components/mgt-file/mgt-file.ts

```diff
@@ -620,17 +628,26 @@ export class MgtFile extends MgtTemplatedComponent {
       switch (current) {
         case 'size':
           // convert size to mb
-          const sizeInMb = (driveItem.size / (1024 * 1024)).toFixed(2);
+          let sizeInMb;
```

Comment:
Thoughts on adding support for KB or GB?

Reply:
Hmm I did this to follow design. It's a good point. Maybe we should. Will look at this shortly.

Reply:
I updated this to convert to kb, mb, gb... but it will be in the mgt-file.ts